### PR TITLE
Add More Browsers Support

### PR DESCRIPTION
**Pull request by skwongr**
* Add browsers support for Audible, BlackBerry, Podbean, ThePodcastApp, TuneIn, Wondery, general Android
* Improve OkHttp

### CODE REVIEW

**Review by demersus — APPROVED** [review]

File: lib/user_agent/browsers/ok_http.rb [hunk 1]
```diff
@@ -59,7 +70,7 @@ def os
       #
       # @return [String]
       def version
-        detect_product(OKHTTP_PRODUCT).version
+        detect { |useragent| OKHTTP_REGEX.match?(useragent.product) }.version
```
Comment (johvet) [inline lib/user_agent/browsers/ok_http.rb:73]: could this not return nil? Would it make sense to have `detect_produc` support a regular expression?

**Review by johvet — CHANGES_REQUESTED** [review]

File: lib/user_agent/browsers/ok_http.rb (hunk 1 above)
Comment (skwongr) [reply lib/user_agent/browsers/ok_http.rb:73]: This is the same check as in `extend?` so would never be nil otherwise wouldn't be used as the browser.

Comment (johvet) [reply lib/user_agent/browsers/ok_http.rb:73]: ok, that's fine by me then

File: lib/user_agent/browsers/wondery.rb [hunk 2]
```diff
@@ -0,0 +1,91 @@
+class UserAgent
+  module Browsers
+    # wondery/android/v1.8.2/2314
+    # Wondery/1660 CFNetwork/1183 Darwin/20.0.0
+    # wondery/ios/v1.8.2/48
+    # wondery/ios/v1.4.2/1 ,AppleCoreMedia/1.0.0.17G64 (iPhone; U; CPU OS 13_6 like Mac OS X; en_us)
```
Comment (johvet) [inline lib/user_agent/browsers/wondery.rb:6]: wondery sometimes uses UAs with staging or development added. Maybe you want to reach out to them in our separate slack channel to confirm this is good enough.

**Review by johvet — APPROVED** [review]

**Pull request merged**